test/fixture/snapshots/test-snapshot-location/package.json

```diff
@@ -0,0 +1,8 @@
+{
+	"ava": {
+		"files": [
+			"src/**/*test.js"
+		],
+	"snapshotDir": "snapshot-fixtures"
+	}
```

Comment:
Nitpick alert: the indentation in this file is inconsistent. It's just a fixture, but once seen I can't not comment 😉